iOS/UI/Reader/Page/CropBordersProcessor.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  CropBordersProcessor.swift
+//  Aidoku (iOS)
+//
+//  Created by Axel Lopez on 20/06/2023.
+//
+
+import Foundation
+import Nuke
+
+#if os(iOS) || os(tvOS)
+import UIKit
+#else
+import CoreGraphics
+#endif
```

Comment:
you only need to import coregraphics
```suggestion
import CoreGraphics
```